content/applications/inventory_and_mrp/inventory/product_management/configure/type.rst

```diff
@@ -4,228 +4,347 @@ Product type
      - No
-     - No
-   * - Create purchase order
+   * - :ref:`Can be included in a purchase order <inventory/product_management/po>`
+     - Yes
+     - Yes
+   * - :ref:`Use putaway rules <inventory/product_management/putaway>`
      - Yes
-     - :ref:`Yes* <inventory/product_management/po>`
-     - No
-   * - Can be manufactured or subcontracted
-     - :ref:`Yes* <inventory/product_management/manufacture>`
-     - :ref:`Yes* <inventory/product_management/manufacture>`
      - No
-   * - Can be in a kit
+   * - :ref:`Can be manufactured, subcontracted, or used in another good's BoM
+       <inventory/product_management/manufacturing>`
      - Yes
      - Yes
+   * - :doc:`Use inventory adjustments <../../warehouses_storage/inventory_management/count_products>`
+     - Yes
      - No
-   * - Placed in package
+   * - :doc:`Use inventory valuation <../inventory_valuation/using_inventory_valuation>`
      - Yes
-     - :ref:`Yes* <inventory/product_management/package>`
      - No
-   * - Appears on Inventory report
-     - :ref:`Yes <inventory/product_management/report>`
+   * - :ref:`Create transfer <inventory/product_management/transfer-store>`
+     - Yes
+     - Yes
+   * - :doc:`Use lot/serial number tracking <../product_tracking>`
+     - Yes
      - No
+   * - :doc:`Can be placed in a kit <../../../manufacturing/advanced_configuration/kit_shipping>`
+     - Yes
+     - Yes
+   * - :ref:`Can be placed in a package <inventory/product_management/package>`
+     - Yes
+     - Yes
+   * - :ref:`Appears on inventory reports <inventory/product_management/report>`
+     - Yes
      - No
 
-.. _inventory/product_management/on-hand-store:
+Inventory
+~~~~~~~~~
 
-On-hand quantity
-----------------
+.. _inventory/product_management/on-hand:
 
-A storable product's on-hand and forecasted quantities, based on incoming and outgoing orders, are
-reflected on the product form, accessed by going to :menuselection:`Inventory app --> Products -->
-Products`, and selecting the desired product.
+On-hand and forecasted quantities
+*********************************
 
-.. figure:: type/on-hand.png
-   :align: center
-   :alt: Show "On Hand" and "Forecast" smart buttons.
+A tracked product's on-hand and forecasted quantities, based on incoming and outgoing orders, are
+reflected on the product form with two smart buttons:
 
-   Current and forecasted quantities are displayed in the **On Hand** and **Forecasted** smart
-   buttons on the product form.
+- :icon:`fa-cubes` :guilabel:`On-Hand Quantity`: This represents the number of units currently
+  available in inventory. Click the button to view or add stock levels for a tracked product.
+- :icon:`fa-chart` :guilabel:`Forecasted`: This represents the number of units *expected* to be
+  available in inventory after all orders are taken into account. In other words,
+  `forecasted = on hand quantity + incoming shipments - outgoing shipments`. Click the button
+  to view the :guilabel:`Forecasted Report`.
```

Comment:
Fun fact! Totally optional to use the [:math: role](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#math) for maths!

(sorry for some reason this suggestion was not added to my last review)

<img width="960" alt="Screenshot 2024-12-03 at 5 57 28 PM" src="https://github.com/user-attachments/assets/63df8b43-5920-4d2e-847b-aed1b4e5ec32">

```suggestion
  :math:`\text{forecasted} = \text{on hand quantity} + \text{incoming shipments} -
  \text{outgoing shipments}`. Click the button to view the :guilabel:`Forecasted Report`.
```